hacs.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "Raspberry Pi GPIO",
-  "homeassistant": "2022.7.0"
+  "homeassistant": "2022.7.0",
+  "render_readme": true
```

Comment:
```suggestion
  "homeassistant": "2022.7.0"
```
`render_readme` is no longer supported by `HACS`

Reply:
done, but dont you want to bump up the version?

Reply:
We will bump it before the release